src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithDefaultNullFW.java

```diff
@@ -0,0 +1,387 @@
+
+    private static final byte NULL_VALUE = 0x40;
+
+    private static final int FIRST_FIELD_OFFSET = LOGICAL_LENGTH_OFFSET + LOGICAL_LENGTH_SIZE;
+
+    private static final int INDEX_VARIANT_OF_STRING1 = 0;
+
+    private static final int INDEX_VARIANT_OF_STRING2 = 1;
+
+    private static final int MASK_VARIANT_OF_STRING2 = 1 << INDEX_VARIANT_OF_STRING2;
+
+    private static final int INDEX_VARIANT_OF_UINT = 2;
+
+    private static final int MASK_VARIANT_OF_UINT = 1 << INDEX_VARIANT_OF_UINT;
+
+    private static final int INDEX_VARIANT_OF_INT = 3;
+
+    private static final int MASK_VARIANT_OF_INT = 1 << INDEX_VARIANT_OF_INT;
+
+    private static final long DEFAULT_VARIANT_OF_UINT = 4000000000L;
+
+    private VariantEnumKindWithString32FW variantOfString1RO = new VariantEnumKindWithString32FW();
+
+    private VariantEnumKindWithString32FW variantOfString2RO = new VariantEnumKindWithString32FW();
+
+    private VariantEnumKindOfUint32FW variantOfUintRO = new VariantEnumKindOfUint32FW();
+
+    private VariantEnumKindWithInt32FW variantOfIntRO = new VariantEnumKindWithInt32FW();
+
+    private long bitmask;
+
+    public int length()
+    {
+        return buffer().getInt(offset() + LOGICAL_LENGTH_OFFSET);
+    }
+
+    public String variantOfString1()
+    {
+        return variantOfString1RO.get();
+    }
+
+    public String variantOfString2()
+    {
+        assert (bitmask & MASK_VARIANT_OF_STRING2) != 0 : "Field \"variantOfString2\" is not set";
+        return variantOfString2RO.get();
+    }
+
+    public long variantOfUint()
+    {
+        return (bitmask & MASK_VARIANT_OF_UINT) != 0 ? variantOfUintRO.get() : DEFAULT_VARIANT_OF_UINT;
+    }
+
+    public int variantOfInt()
+    {
+        assert (bitmask & MASK_VARIANT_OF_INT) != 0 : "Field \"variantOfInt\" is not set";
+        return variantOfIntRO.get();
+    }
+
+    @Override
+    public ListWithDefaultNullFW wrap(
+        DirectBuffer buffer,
+        int offset,
+        int maxLimit)
+    {
+        super.wrap(buffer, offset, maxLimit);
+        int fieldLimit = offset + FIRST_FIELD_OFFSET;
+        bitmask = 0;
+        final int length = length();
+        for (int field = INDEX_VARIANT_OF_STRING1; field < length; field++)
+        {
+            switch (field)
+            {
+            case INDEX_VARIANT_OF_STRING1:
+                variantOfString1RO.wrap(buffer, fieldLimit, maxLimit);
+                fieldLimit = variantOfString1RO.limit();
+                bitmask |= 1 << INDEX_VARIANT_OF_STRING1;
+                break;
+            case INDEX_VARIANT_OF_STRING2:
+                if (buffer().getByte(fieldLimit) != NULL_VALUE)
+                {
+                    variantOfString2RO.wrap(buffer, fieldLimit, maxLimit);
+                    fieldLimit = variantOfString2RO.limit();
+                    bitmask |= 1 << INDEX_VARIANT_OF_STRING2;
+                }
+                else
+                {
+                    fieldLimit++;
```

Comment:
Hoist constant `private final int NULL_VALUE_SIZE = BitUtil.SIZE_OF_BYTE;` and use here via `fieldLimit += NULL_VALUE_SIZE`.